packages/app/app/containers/BestNewMusicCardContainer/BestNewMusicCardContainer.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { waitFor } from '@testing-library/react';
+import pitchforkBnm from 'pitchfork-bnm';
+import { Deezer } from '@nuclear/core/src/rest';
+
+import { buildStoreState } from '../../../test/storeBuilders';
+import { mountedComponentFactory, setupI18Next } from '../../../test/testUtils';
+
+jest.mock('pitchfork-bnm');
```

Comment:
To mock particular functions in this import, you can use `mockImplementation`.

Reply:
I think this will also help avoid the need to use @ts-ignore comments.

Reply:
Have made the mockImplementation change - does it look right? Tests are happy.

Reply:
Perfect!